flyteidl/protos/flyteidl/core/literals.proto

```diff
@@ -102,6 +102,9 @@ message Literal {
 
         // A map of strings to literals.
         LiteralMap map = 3;
+
+        // Offloaded literal metadata
```

Comment:
when i deserialize the offloaded data, what do I get back?  a `Literal` message?  or do I get one of a `Scalar`, `LiteralCollection`, `LiteralMap` (i'm assuming the former but can you add a comment?

Reply:
It depends on the LiteralType defined in the offloaded message https://github.com/flyteorg/flyte/pull/5705/files#diff-94ec167526f42a9db5cd35d1b1807676c5e791f407b5a9e93193acbdffe6f3f7R128

let me add a comment